editor/block-mover/style.scss

```diff
@@ -3,7 +3,6 @@
 	top: 10px;
 	left: 0;
 	padding: 0 14px 20px 0; // handles hover area
-	z-index: z-index( '.editor-block-mover' );
```

Comment:
If we're to remove this, the corresponding entry should also be removed from the `editor/assets/stylesheets/_z-index.scss` file.